packages/atomic/src/components/search/result-template-components/atomic-quickview/e2e/atomic-quickview.e2e.ts

```diff
@@ -144,7 +144,6 @@ test.describe('Quickview', () => {
       await quickview.load();
       await page.setViewportSize({width, height});
       await quickview.resultButton.click();
-      await quickview.resultButton.click();
       await quickview.modal.waitFor({state: 'visible'});
```

Comment:
default `waitFor` is visible, so you don't have to input the option

Reply:
Yea I am just correcting a test that was failing locally for me because of the double click, this isn't my test